doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+      :alt: LTE Link Monitor - Automatic requests check box
+
+#. If you have already inserted the SIM card into your DK, remove it before you continue.
+#. Connect the nRF9160 DK to the computer with a micro-USB cable, and turn it on.
+#. Click :guilabel:`Select device` and select the DK from the drop-down list.
+
+   .. figure:: /images/ltelinkmonitor_selectdevice.png
+      :alt: LTE Link Monitor - Select Device
+
+      LTE Link Monitor - Select Device
+
+#. Click :guilabel:`Terminal` in the navigation bar to switch to the terminal view.
+
+   .. figure:: /images/ltelinkmonitor_navigationterminal.png
+      :alt: LTE Link Monitor - Terminal
+
+      LTE Link Monitor - Terminal
+
+#. Enter ``AT+CFUN=4`` in the text field for AT commands and click :guilabel:`Send`.
+   This AT command puts the modem to offline state.
+#. Enter ``AT+CFUN?`` in the text field for AT commands and click :guilabel:`Send`.
+   This AT command returns the state of the modem.
+
+   The command should return ``+CFUN: 4``, which indicates that the modem is in offline state.
+   If it returns a different value, repeat the previous step.
+#. Click :guilabel:`Certificate manager` in the navigation bar to switch to the certificate manager view.
+
+   .. figure:: /images/ltelinkmonitor_navigationcertificatemanager.png
+      :alt: LTE Link Monitor - Certificate Manager
+
+      LTE Link Monitor - Certificate Manager
+
+#. Click :guilabel:`Load from JSON` and select the :file:`*.cert.json` file tat you downloaded from nRF Cloud.
+   Alternatively, you can drag and drop the file onto the GUI.
+
+   .. figure:: /images/ltelinkmonitor_loadjson.png
+      :alt: LTE Link Monitor - Load from JSON
+
+      LTE Link Monitor - Load from JSON
+
+#. Ensure that the :guilabel:`Security tag` is set to ``16842753``, which is the security tag for nRF Cloud credentials.
+#. Click :guilabel:`Update certificate`.
+
+   .. figure:: /images/ltelinkmonitor_updatecertificates.png
+      :alt: LTE Link Monitor - Update certificates
+
+      LTE Link Monitor - Update certificates
+
+   The log message "Certificate update completed" indicates that the certificate was provisioned successfully.
+   If you encounter any errors, switch to the terminal view and check the output of the AT commands that were sent to the nRF9160 DK modem.
+
+   .. note::
+
+      If you had connected your nRF9160 DK to nRF Cloud before, you must delete the device there after provisioning the certificate.
+      To do so, open the entry for your device from the :guilabel:`Devices` view, then click the gear icon to the right of the device's name, and select :guilabel:`Delete Device`.
+      Then, add the nRF9160 DK again as described in :ref:`nrf9160_gs_connecting_dk_to_cloud`
+
+.. _nrf9160_gs_connecting_dk_to_cloud:
+
+Connecting the DK to nRF Cloud
+******************************
+
+To transmit data from your nRF9160 DK to nRF cloud, you must activate your SIM card and add the nRF9160 DK to your nRF Cloud account.
```

Comment:
SIM card activation is only needed once per iBasis SIM, and not for other SIMs or if an iBasis SIM has previously been used and activated with another device

Reply:
Moved note from later in the section to here to make this more clear.